sdk/communication/azure-communication-identity/swagger/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <name>Microsoft Azure Communication Services Identity client for Java</name>
+  <description>This package contains client functionality for Microsoft Azure Communication Services Identity</description>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>8</source>
+          <target>8</target>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-communication-identity</artifactId>
+  <version>1.2.0-beta.2</version>
+  <packaging>jar</packaging>
```

Comment:
CI is failing as this groupId, artifactId, and version match the exact identifier for the `azure-communication-identity` SDK which is causing tooling to ignore the actual project. Could you do two things here, change the groupdId to `com.azure.tools` and `azure-communication-identity-autorest-customization`.